SPECS/nexus3-oss.spec

```diff
@@ -170,6 +174,10 @@ rm -rf $RPM_BUILD_ROOT
 %endif
 
 %changelog
+* Thu Oct 12 2023 Julio González Gil <packages@juliogonzalez.es> - 3.61.0.02-2
```

Comment:
Happy to provide credit here, if you tell me what email you want to use :-)

Reply:
opensuse@jdsn.de

Reply:
Done!